### PR TITLE
Add simple output.

### PR DESCRIPTION
**Pull request by mgsergio**
@ygorshenin @bykoianko

### CODE REVIEW

File: base/base_tests/newtype_test.cpp [hunk 1]
```diff
@@ -96,4 +96,19 @@ UNIT_TEST(NewTypeMember_Operations)
   a ^= Int(7);
   TEST_EQUAL(a, Int(10 ^ 7), ());
 }
+
+namespace test_output
```
Comment (ygorshenin) [inline base/base_tests/newtype_test.cpp]: No need in namespace --- all this stuff in the anonymous namespace already.

File: base/newtype.hpp [hunk 2]
```diff
@@ -135,8 +140,27 @@ class NewType
 private:
   Type m_value;
 };
+
+namespace newtype_default_output
+{
+template <typename Type, typename Tag>
+inline string SimpleDebugPrint(NewType<Type, Tag> const & nt)
```
Comment (ygorshenin) [inline base/newtype.hpp]: I guess inline is superfluous here.

File: base/base_tests/newtype_test.cpp (hunk 1 above)
Comment (mgsergio) [inline base/base_tests/newtype_test.cpp]: I need this, cause I have to different DebugPrints: one from above and one defined by a macro.

File: base/newtype.hpp [hunk 3]
```diff
@@ -135,8 +140,27 @@ class NewType
...
+{
+  return DebugPrint(nt.Get());
+}
+}  // namespace newtype_default_output
 }  // namespace my
 
 #define NEWTYPE(REPR, NAME)                     \
   struct NAME ## _tag;                          \
-  using NAME = my::NewType<REPR, NAME ## _tag>
+  using NAME = my::NewType<REPR, NAME ## _tag>  \
```
Comment (ygorshenin) [inline base/newtype.hpp]: No need in slash here.

File: base/newtype.hpp [hunk 4]
```diff
@@ -135,8 +140,27 @@ class NewType
...
+
+#define NEWTYPE_SIMPLE_OUTPUT(NAME)                                     \
+  inline string DebugPrint(NAME const & nt)                             \
+  {                                                                     \
+    return my::newtype_default_output::SimpleDebugPrint(nt);            \
+  }                                                                     \
+  inline ostream & operator<<(ostream & ost, NAME const & nt)           \
+  {                                                                     \
+    return ost << my::newtype_default_output::SimpleDebugPrint(nt);     \
+  }                                                                     \
```
Comment (ygorshenin) [inline base/newtype.hpp]: No need in stlash here.

File: base/base_tests/newtype_test.cpp (hunk 1 above)
Comment (ygorshenin) [inline base/base_tests/newtype_test.cpp]: OK

File: base/newtype.hpp (hunk 4 above)
Comment (mgsergio) [inline base/newtype.hpp]: Done.

File: base/newtype.hpp (hunk 3 above)
Comment (mgsergio) [inline base/newtype.hpp]: Done.

File: base/newtype.hpp (hunk 2 above)
Comment (mgsergio) [inline base/newtype.hpp]: Done.

**Comment by mgsergio** [comment]
PTAL

**Comment by ygorshenin** [comment]
LGTM

**Comment by bykoianko** [comment]
LGTM

**Pull request merged**